lib/components/atoms/Popover/Popover.mock.js

```diff
@@ -0,0 +1,7 @@
+const defaultConfig = {
```

Comment:
Can you also make a sample where we are putting another component like LIst inside popover?